examples/Makefile-nrf52840

```diff
@@ -85,10 +85,23 @@ COMMONCFLAGS                      := \
 include $(dir $(abspath $(lastword $(MAKEFILE_LIST))))/common-switches.mk
 
 ifeq ($(BOOTLOADER),1)
-configure_OPTIONS += --with-custom-linker-file=$(AbsTopSourceDir)/examples/platforms/nrf528xx/nrf52840/nrf52840_bootloader.ld
+$(info Warning: BOOTLOADER switch is deprecated. Defaulting to BOOTLOADER_USB=1.)
```

Comment:
Maybe log as a warning? It shouldn't change much but `info Warning` looks strange.

Reply:
The only thing that `warning` does as an extra is print the line number. I don't think that is needed.